com.moddb.TotalChaosRetro.yaml

```diff
@@ -0,0 +1,94 @@
+app-id: com.moddb.TotalChaosRetro
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: gzdoom.sh
+finish-args:
+  - --device=all
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --env=DOOMWADDIR=/app/share/games/doom
+  - --persist=.config/gzdoom
+cleanup:
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+  - /lib/pkgconfig
+  - /share/man
+modules:
+  - name: fluidsynth
```

Comment:
you should use the shared-module instead.

Reply:
Good idea, I should probably also switch to using [the shared-module](https://github.com/flathub/shared-modules/blob/master/linux-audio/fluidsynth2.json) for fluidsynth in my Total Chaos Flatpak. :-)

Reply:
@AsciiWolf can you help me to do this?

Reply:
Just type:
`git submodule add https://github.com/flathub/shared-modules.git`

and replace the fluidsynth module with:
`  - shared-modules/linux-audio/fluidsynth2.json`

Then git commit with `-am`.

Reply:
Just type:
`git submodule add https://github.com/flathub/shared-modules.git`

and replace the fluidsynth module with:
`  - shared-modules/linux-audio/fluidsynth2.json`

Then git commit with `-am`.

Reply:
@hfiguiere done, thanks @AsciiWolf